src/main/java/core/basesyntax/serviceimpl/WriterServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.WriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void writeToFile(String filePath,List<String> lines) {
```

Comment:
Please ensure that the method parameter 'List<String> lines' is properly documented with JavaDoc to explain what this list represents and any requirements or expectations for its contents.